v21.1/connect-to-the-database-cockroachcloud.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Connect to the Database (CockroachCloud)
+summary: How to connect to a CockroachCloud cluster from your application
+toc: true
+---
+
+<div class="filters filters-big clearfix">
+  <a href="connect-to-the-database.html"><button class="filter-button">Local</button></a>
+  <button class="filter-button current">CockroachCloud</button>
+</div>
+
+This page has instructions for connecting to a CockroachCloud cluster from your application using various programming languages. Each example shows a [connection string][connection_params] for a secure cluster to a `bank` database. Depending on your cluster's configuration, you may need to edit this connection string.
+
+The connection strings listed on this page set the required authentication options to connect to [free CockroachCloud](cockroachcloud/authentication.html) clusters. CockroachCloud clusters use a signed certificate generated for your cluster that you download from the CockroachCloud console.
```

Comment:
> cockroachcloud/authentication.html

Any refs that start with `cockroachcloud` are going to break, because that directory is up one from the version-specific CockroachDB docs. `cockroachcloud/authentication.html` is generated as `{version}/cockroachcloud/authentication.html`. All CockroachCloud doc refs need to start with `../cockroachcloud`.